configs/queries/analyze-repo-top-contributors/template.sql

```diff
@@ -0,0 +1,23 @@
+WITH contributions AS (
+    SELECT
+        actor_login, COUNT(*) AS events
```

Comment:
You don't need to use CTE.

```suggestion
        actor_login, COUNT(*) AS engagements
```